lib/IAMClient.js

```diff
@@ -178,15 +178,15 @@ class VaultClient {
         };
         if (typeof options.marker !== 'undefined') {
             const marker = options.marker;
-            assert.notStrictEqual(isNaN(marker) || marker === '', true,
+            assert.notStrictEqual(Number.isNaN(marker) || marker === '', true,
```

Comment:
Test only breaks with this change 🤔 
Change of `Number.isNaN` below at L189 does not break tests

Reply:
TIL `Number.isNaN` is different from how `isNaN` works:
https://stackoverflow.com/questions/33164725/confusion-between-isnan-and-number-isnan-in-javascript